torch_ort_inference/demos/notebooks/roberta-base-squad2_for_queston_answering/roberta_base_squad2_for_queston_answering.ipynb

```diff
@@ -0,0 +1,354 @@
+        "1. This demo shows how to use Intel® OpenVINO™ integration with Torch-ORT to extract the answer from a given context based on the questions.\n",
+        "\n",
+        "2. We use a question answering model deepset/roberta-base-squad2 from HuggingFace models. This model is trained on the RoBERTa architecture to do extractive question answering."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "kglwHDDR1tcS"
+      },
+      "source": [
+        "### Import Necessary Resources"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 41,
+      "metadata": {
+        "id": "OOtdydIbzdC0"
+      },
+      "outputs": [],
+      "source": [
+        "import os\n",
+        "import numpy as np\n",
+        "import time\n",
+        "import pandas as pd\n",
+        "import pathlib\n",
+        "\n",
+        "from transformers import AutoModelForQuestionAnswering, AutoTokenizer\n",
+        "\n",
+        "import torch\n",
+        "from torch_ort import ORTInferenceModule, OpenVINOProviderOptions"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XNRDByffS1Ox"
+      },
+      "source": [
+        "### Preprocess Function\n",
+        "\n",
+        "* Use auto tokenizer for question answering to preprocess the inputs"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 42,
+      "metadata": {
+        "id": "VBN3gCJL2Xm0"
+      },
+      "outputs": [],
+      "source": [
+        "def preprocess_input(tokenizer, questions, context):\n",
+        "  inputs = []\n",
+        "  for question in questions:\n",
+        "    inputs.append(tokenizer(question, context, add_special_tokens=True, return_tensors=\"pt\"))\n",
+        "  return inputs"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mrpWASQYSulH"
+      },
+      "source": [
+        "### Inference Function\n",
+        "* Runs the inference on the input sentences\n",
+        "* Prints the inference results"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 43,
+      "metadata": {
+        "id": "tXNAK_rn2YI6"
+      },
+      "outputs": [],
+      "source": [
+        "def infer(model, tokenizer, questions, inputs):\n",
+        "    num_questions = len(questions)\n",
+        "    total_infer_time = 0\n",
+        "    results = {}\n",
+        "\n",
+        "\n",
+        "    # Run inference\n",
+        "    for i in range(num_questions):\n",
+        "        input_ids = inputs[i][\"input_ids\"].tolist()[0]\n",
+        "        with torch.no_grad():\n",
+        "            # warm-up\n",
+        "            if i==0:\n",
+        "               t0 = time.time()\n",
+        "               outputs = model(**inputs[i])\n",
+        "            # infer\n",
+        "            t0 = time.time()\n",
+        "            outputs = outputs = model(**inputs[i])\n",
```

Comment:
this probably should be `outputs = model(...)`

Reply:
Good catch. Fixed the typo.